ocfweb/docs/docs/about.md

```diff
@@ -30,21 +31,24 @@ the ground floor of MLK Student Union.
 To help people explore the field of computing and educate them about the
 resources available, the OCF also provides a variety of educational services.
 We used to co-sponsor a series of Help Sessions, short classes taught
-throughout the semester on a wide range of computing topics, from beginner to
-advanced levels, and focusing on resources available to the Berkeley campus
-community.  These now take the form of [DeCal courses][decal] offered by us. We
-also offer e-mail and in-person consulting with our volunteer staff members, as
-well as online sources of information about computing on the Berkeley campus
-and how to use the available resources.  Additionally, the OCF has conducted
-career fairs in the past with its sister organization, the [Computer Science
-Undergraduate Association][csua].
+throughout the semester on a wide range of computing topics (focusing on
+resources available to the Berkeley campus community), along with hosting the
+[UNIX SysAdmin DeCal][decal]. Additionally, the OCF used to conduct
+career fairs in the past with its sister organization, the
+[Computer Science Undergraduate Association][csua]. Now, we host regular Tech
+Talks that explore new technologies and how they might  to elements of the
+OCF infrastructure. We also offer e-mail and in-person with our volunteer
+staff members, as well as online sources of information about computing on the
+Berkeley campus and how to use the available resources.
 
 Policies for this cluster and decisions for the OCF organization are made by
-the OCF Board of Directors, a group of students elected each semester by the
-OCF membership.  The Board meetings, held at least once every two weeks when
-classes are in session, are open to the public and provide a chance for any
-user to express their concerns or opinions about the OCF. [Minutes of previous
-meetings][minutes] are also available online.
+the OCF Board of Directors, an open body composed of OCF members who are
+interested in participating in the decision-making process. The Board
+convenes at BoD Meetings, held at least every week when classes are in session,
```

Comment:
Yeah, I removed the every other week thing and replaced it with "at least every week." Are you saying we should remove "at least"?